src/apis/queryKeys.ts

```diff
@@ -32,6 +32,7 @@ export const QUERY_KEYS = {
 
   users: {
     get: 'users',
+    edit: 'myInfomations',
```

Comment:
함수를 사용하여 동적으로 생성하면 어떨까요~?
```suggestion
    edit: (userId: number) => ['myInfomations', userId]
```